src/ProjectOrigin.WalletSystem.Server/Extensions/IEnumerableExtensions.cs

```diff
@@ -6,4 +6,9 @@ namespace ProjectOrigin.WalletSystem.Server.Extensions;
 public static class IEnumerableExtensions
 {
     public static bool IsEmpty<TSource>(this IEnumerable<TSource> source) => !source.Any();
+
+    public static IEnumerable<T> NotNull<T>(this IEnumerable<T?> enumerable) where T : class
```

Comment:
From the name `NotNull` I would expect this to return a bool similar to `IsEmpty`. Changing this to e.g. `FilterNulls` or `RemoveNulls` or `PruneNulls` would make it more clear.

Reply:
no longer used since rewrite.

Reply:
Yup, that is beter